arwen/contexts/runtime.go

```diff
@@ -518,7 +518,12 @@ func (context *runtimeContext) FailExecution(err error) {
 
 	var message string
 	if err != nil {
-		message = err.Error()
+		wrappedError, ok := (err.(arwen.WrappableError))
```

Comment:
can be simplified as:
```
wrappedError, ok := err.(arwen.WrappableError)
```

Reply:
err is not always a WrappableError if this is the assumption

Reply:
the paranthesis are duplicated.

Reply:
no, please remove the redundant parenthesis around the type assertion. It raises linter issues.